scripts/bin/bb-build

```diff
@@ -35,7 +37,8 @@ def parse_spec(spec_file):
 @click.option('--output', default='sampler', help='Sampler executable name.')
```

Comment:
While at it, I think we can change the help message to something more informative.

Reply:
Good point, I hadn't seen that